Documentation/SwiftGenKit Contexts/CoreData.md

```diff
@@ -0,0 +1,264 @@
+# Core Data parser
+
+## Input
+
+The core data parser accepts a `xcdatamodeld` (or `xcdatamodel`) file. The parser will load all the configurations, entities, and fetch requests for each model.
+
+## Output
+
+The output context has the following structure:
+
+ - `models`: `Array` — A list of parsed models, with each:
+   - `configurations`: `Dictionary<Array<String>>` — Map of configurations and the corresponding list of entity names
+   - `entities`: `Dictionary` — Map of all the entities in the model, each key being the entity's name:
```

Comment:
"… and the corresponding value being a sub-dictionary with the following keys:"

(Seeing how convoluted that sentence might seem for the reader, maybe separating the sub-dictionary in its own paragraph will be clearer after all?

eg
```
 - `entities`: `Dictionary` – Map of all the entities in the model, each key being the entity's name, and corresponding value is a dictionary described below
```